web/public/mgmt/config.yml

```diff
@@ -1784,22 +1784,71 @@ collections:
       preview: true
     fields:
-      - { label: "Content", name: "body", widget: "markdown" }
-      - { label: "Url Slug", name: "urlSlug", widget: "nospace" }
       - { label: "Filename", name: "filename", widget: "nospace", required: false }
-      - { label: "Name", name: "name", widget: "string" }
-      - { label: "Id", name: "id", widget: "nospace" }
-      - { label: "Call To Action Link", name: "callToActionLink", widget: "string", required: false }
-      - { label: "Call To Action Text", name: "callToActionText", widget: "string", required: false }
-      - { label: "Industry", name: "industryId", widget: "string", required: false }
-      - { label: "Issuing Agency", name: "issuingAgency", widget: "string", required: false }
-      - { label: "Issuing Division", name: "issuingDivision", widget: "string", required: false }
-      - { label: "Division Phone", name: "divisionPhone", widget: "string", required: false }
+      - { label: "Url Slug", name: "urlSlug", widget: "nospace" }
+      - { label: "Title (Navigator)", name: "name", widget: "string" }
+      - { label: "Title (Webflow)", name: "webflowName", widget: "string" }
+      - { label: "Content", name: "body", widget: "markdown" }
+      - {
+          label: "State Agency (Webflow & Navigator)",
+          name: "industryIntegration",
+          widget: "select",
+          required: false,
+          options:
+            [
+              "New Jersey Division of Consumer Affairs",
+              "New Jersey Department of Taxation",
+              "New Jersey Department of Revenue and Enterprise Services",
+              "New Jersey Department of Community Affairs",
+              "New Jersey Department of Banking and Insurance",
+              "New Jersey Department of Health",
+            ],
+        }
+      - {
+          label: "Local agency (Webflow & Navigator)",
+          name: "localLevelTask",
+          widget: "select",
+          required: false,
+          options: ["Local Department of Health", "County Clerk", "Municipal Clerk"],
+        }
       - {
-          label: "License Certification Classification",
+          label: "Agency additional context (Webflow & Navigator)",
+          name: "issuingAgency",
+          widget: "string",
+          required: false,
+        }
+      - { label: "Form Name (Webflow & Navigator)", name: "formName", widget: "string", required: false }
+      - { label: "Industry Data From Webflow", name: "industryId", widget: "string", required: false }
+      - {
+          label: "Call To Action Text (Webflow & Navigator)",
+          name: "callToActionText",
+          widget: "string",
+          required: false,
+        }
+      - {
+          label: "Call To Action Link (Webflow & Navigator)",
+          name: "callToActionLink",
+          widget: "string",
+          required: false,
+        }
+      - { label: "Industry (Webflow - archive)", name: "webflowIndustry", widget: "string", required: false }
```

Comment:
So I was checking whether fields already existed via the name field. So I assumed that name: "industryId" between the 2 was the one we were talking about this morning for consolidation. So that field was initially the drop down but was replaced with a single field.

But I think I understand what you're saying and I'll try to implement later today after my meetings and appointments.

Also come to think of it I'm not sure I completely understand how / what this is used for from a users perepctive and that might be good to know. So I'll ask later